andela/mervels-bn-backend
# #168131409 Setup API documentation using swagger

#### What does this PR do?
Implement swagger documentation
#### Description of Task to be completed?
- adds the swagger-UI-express module
- adds the swagger.json file and route
- adds babelrc file
- fixes babelrc dependency issues
#### How should this be manually tested?
- make sure you have MongoDB installed
- Run npm install
- Run npm start for production or npm start: dev for development
- Go to the browser and head over to `localhost:3000/api/docs`

#### What are the relevant pivotal tracker stories?
[#168131409](https://www.pivotaltracker.com/story/show/168131409)
#### Screenshots (if appropriate)
<img width="1436" alt="Screen Shot 2019-09-04 at 08 51 35" src="https://user-images.githubusercontent.com/26383328/64232263-ebcc8480-cef1-11e9-8e03-5876fc1795da.png">

#### Questions:
N/A
